### ISSUE TITLE
[New rule] avoid void arrow function

### ISSUE THREAD

**Issue by Mahakon** [dart-code-checker/dart-code-metrics#156]
For dart code is better to avoid use void arrow function, for several reasons: 

- after compilation ```return``` is added to such function:
```dart
void funcA() => a += 55;
```
```javascript
function(){return $.a=$.a+55}
```

- if in such function we add more action in git diff it looks better 

**Please describe what the rule should do:**
it will ask not to use void arrow function 

**If your rule is inspired by other please provide link to it:**
-

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[X] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

```dart
 //bad
void funcA() => a += 55;

//good
void funcA() {
  a += 55;
}
```

**Are you willing to submit a pull request to implement this rule?**
yes

**Comment by incendial** [comment]
We decided not to implement it due to a small impact and potential usage.
